lib/modules/manager/tekton/readme.md

```diff
@@ -25,6 +25,24 @@ There are three ways to use a Tekton Bundle reference:
 
 Renovate's Tekton manager supports all the methods listed above.
 
+### Using image in Tekton Tasks
+
+Tekton Tasks define container images that are executed during its execution.
+There are several attributes where such images can be defined:
+
+1. In a Task step.
+1. In a Task stepTemplate.
+1. In a Task sidecar.
+
+Furthermore, Tekton Task definitions can be defined within various Tekton resources:
```

Comment:
Nice!

I remove the "also" from your suggestion because so far the doc doesn't actually mention where the you can define Tekton Tasks. Let me know what you think!